test/extended/README.md

```diff
@@ -33,6 +36,67 @@ $ go get github.com/onsi/ginkgo/ginkgo
 You also need to have the `openshift` binary in the `PATH` if you want to use
 the shell script helpers to execute the extended tests.
 
+Rapid local testing
+--------------------
+
+If you already have a running OpenShift cluster, e.g. one created using `oc
+cluster up`, you can skip having the extended test infrastructure spin up an
+OpenShift cluster each time the tests are run by setting the `TEST_ONLY`
+environment variable as follows:
```

Comment:
Hmm -- inside of [`cleanup_openshift`](https://github.com/openshift/origin/blob/master/hack/util.sh#L86-L91) we kill Docker containers on the host daemon that match the k8s naming convention. To be safe, I'd say we should suggest that users set both `TEST_ONLY` and `SKIP_CLEANUP` -- maybe this doesn't break `oc cluster up` but it will break manual `openshift start` type scenarios like what we use for the conformance test setup.